OpenSim/Examples/CustomActuatorExample/PistonActuator.h

```diff
@@ -121,38 +112,33 @@ OpenSim_DECLARE_CONCRETE_OBJECT(PistonActuator, ScalarActuator);
     bool getPointsAreGlobal() {return get_points_are_global(); };
 
     // OPTIMAL FORCE
+    /** The force applied by this actuator is its control signal multiplied
+     * by this optimal force. */
     void setOptimalForce(double aOptimalForce);
     double getOptimalForce() const override;
-    // STRESS
-#ifndef SWIG
-    double getStress( const SimTK::State& s ) const override;
+    /** This is the force generated by this actuator divided by its optimal
+     * force. */
+    double getStress(const SimTK::State& s) const override;
```

Comment:
It looks like this function uses an absolute value in it, and the comments in this class clearly state what the sign means for the force computed.  I'm not certain if a user would expect this to only return positive values?

Reply:
It seems like the `fabs()` is unnecessary. However, `PathActuator` also has an `fabs()`:

https://github.com/opensim-org/opensim-core/blob/f721ab592a60b88fde6eb059c3d73d21400879d7/OpenSim/Simulation/Model/PathActuator.cpp#L123-L131

So I modified the comment to mention that there's an absolute value.